src/main/java/org/jabref/logic/exporter/EndnoteXmlExporter.java

```diff
@@ -0,0 +1,218 @@
+        writeField(xml, "number", entry.getField(StandardField.NUMBER).orElse(""), null);
+        writeField(xml, "dates", null, Map.of(), entry.getField(StandardField.YEAR).orElse(""), "year");
+        writeField(xml, "publisher", entry.getField(StandardField.PUBLISHER).orElse(""), null);
+        writeField(xml, "isbn", entry.getField(StandardField.ISBN).orElse(""), null);
+        writeField(xml, "abstract", entry.getField(StandardField.ABSTRACT).orElse(""), null);
+        writeField(xml, "notes", entry.getField(StandardField.NOTE).orElse(""), null);
+        writeField(xml, "urls", null, Map.of(), entry.getField(StandardField.URL).orElse(""), "web-urls");
+        writeField(xml, "electronic-resource-num", entry.getField(StandardField.DOI).orElse(""), null);
+
+        xml.writeEndElement(); // record
+    }
+
+    private void writeField(XMLStreamWriter xml, String name, String value, Map<String, String> attributes) throws XMLStreamException {
+        if (value != null && !value.isEmpty()) {
+            xml.writeStartElement(name);
+            if (attributes != null) {
+                for (Map.Entry<String, String> attribute : attributes.entrySet()) {
+                    xml.writeAttribute(attribute.getKey(), attribute.getValue());
+                }
+            }
+            xml.writeStartElement("style");
+            xml.writeAttribute("face", "normal");
+            xml.writeAttribute("font", "default");
+            xml.writeAttribute("size", "100%");
+            xml.writeCharacters(value);
+            xml.writeEndElement(); // style
+            xml.writeEndElement(); // name
+        }
+    }
+
+    private void writeField(XMLStreamWriter xml, String name, String value, Map<String, String> attributes, String childValue, String childElementName) throws XMLStreamException {
+        xml.writeStartElement(name);
+        if (attributes != null) {
+            for (Map.Entry<String, String> attribute : attributes.entrySet()) {
+                xml.writeAttribute(attribute.getKey(), attribute.getValue());
+            }
+        }
+        if (childValue != null && !childValue.isEmpty()) {
+            xml.writeStartElement(childElementName);
+            xml.writeStartElement("style");
+            xml.writeAttribute("face", "normal");
+            xml.writeAttribute("font", "default");
+            xml.writeAttribute("size", "100%");
+            xml.writeCharacters(childValue);
+            xml.writeEndElement(); // style
+            xml.writeEndElement(); // childElementName
+        }
+        xml.writeEndElement(); // name
+    }
+
+    private void writeContributors(BibEntry entry, XMLStreamWriter xml) throws XMLStreamException {
+        Optional<String> authors = entry.getField(StandardField.AUTHOR);
+        if (authors.isPresent()) {
+            xml.writeStartElement("contributors");
+            xml.writeStartElement("authors");
+            for (String author : authors.get().split("and")) {
```

Comment:
You should not rely on simply splitting by and. You need to use the AuthorLIstParser, see an example here

https://github.com/JabRef/jabref/blob/a902524861dc7975ec98b62a2dffcd818192380b/src/main/java/org/jabref/logic/msbib/MSBibConverter.java#L145-L147 

Author class itself has methods for getting given (first) or lastname 

The Author class has methods like

Reply:
Done. This solved the author name issue:
![image](https://github.com/JabRef/jabref/assets/74734844/e5e39246-cb21-4fd6-858e-0b91dd9d6f02)